common.minimal.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-define('UNIENGINE_LANGS_AVAILABLE', [ 'en', 'pl' ]);
+define('UNIENGINE_LANGS_AVAILABLE', [ 'en', 'pl', 'fr' ]);
```

Comment:
For the time being, I think this should be removed to prevent anyone from accidentally using an incomplete translation. Anyone who will work on the translation progress should be able to change this in their local environment.